src/modules/AieraChat/components/Header/Title/index.tsx

```diff
@@ -0,0 +1,39 @@
+import debounce from 'lodash.debounce';
+import React, { useCallback, useEffect } from 'react';
+import { useChatStore } from '../../../store';
+
+export function Title({ onChangeTitle }: { onChangeTitle: (title: string) => void }) {
+    const { chatTitle, onSetTitle } = useChatStore();
+
+    // Debounce calling the mutation with each change
+    const debouncedTitleChange = useCallback(
+        debounce((title: string) => {
+            onChangeTitle(title);
+        }, 300),
+        [onChangeTitle]
+    );
+
+    // Cleanup the debounced function on component unmount
+    useEffect(() => {
+        return () => {
+            debouncedTitleChange.cancel();
+        };
+    }, [debouncedTitleChange]);
+
+    return (
+        <div className="flex-1 flex items-center text-base font-bold">
+            <input
+                aria-label="Chat title input"
```

Comment:
Screen readers already announce an element's role when reading their label, so this would read "chat title input, input":

```suggestion
                aria-label="Chat title"
```